lib/chef/knife/bootstrap_windows_base.rb

```diff
@@ -199,20 +199,46 @@ def bootstrap_command
         @bootstrap_command ||= "cmd.exe /C #{bootstrap_bat_file}"
       end
 
-      def create_bootstrap_bat_command(&block)
-        bootstrap_bat = []
+      def bootstrap_render_banner_command(chunk_num)
+        "cmd.exe /C echo Rendering #{bootstrap_bat_file} chunk #{chunk_num}"
+      end
+
+      def create_bootstrap_bat_command()
         chunk_num = 0
+        bootstrap_bat = ""
+        banner = bootstrap_render_banner_command(chunk_num += 1)
         render_template(load_template(config[:bootstrap_template])).each_line do |line|
           # escape WIN BATCH special chars
+          # TODO: The commands are going to get redirected - do we need to escape &?
           line.gsub!(/[(<|>)^]/).each{|m| "^#{m}"}
```

Comment:
Good call.  Sometimes I get lazy and I appreciate someone poking me to do the right thing :)